### ISSUE TITLE
High Severity Code Smells: Adaptability Clean Code Attribute

### ISSUE THREAD

**Issue by Tensa53** [Tensa53/commons-net#3]
There are 72 High Severity Code Smells related to the Adaptability Clean Code Attribute, as scanned by SonarCloud. This issue is fixed by merging an appropriate pull request with all bugs fixed. Work is split between the two team members.

Three types of code smells:
- Refactoring a method to reduce its cognitive complexity: 43
- Defining a constant instead of duplicating a literal: 6
- Adding at least one assertion to a test case: 23

**Comment by FraPCA** [comment]
https://github.com/Tensa53/commons-net/commit/43ed18a459bd9f18ece35af17327295605a40d1a
fixes 3/6 of : Defining a constant instead of duplicating this literal
and 12/23 of: Adding at least one assertion to this test case:

Please check the commit message if there are any problems or uncertainties.

**Comment by FraPCA** [comment]
https://github.com/Tensa53/commons-net/commit/3c8e88cf54875d4f4f8e7ce5a0c84ec4ad90b36a , https://github.com/Tensa53/commons-net/commit/b77948aeda7a54b97ad61269cb4ab8848e6b1a03 , https://github.com/Tensa53/commons-net/commit/c88e98a96438d96590f4b2d35194773e3f665218 , https://github.com/Tensa53/commons-net/commit/388eff0783330a269369ff3f6dd508cb659f87c6

fix 22/43 of: Refactoring a method to reduce its cognitive complexity.

There were no problems found during the merging process.

**Comment by FraPCA** [comment]
After careful consideration, we have decided to revert the fixes made in relation to: Adding at least one assertion to a test case.

The original intent by the function authors should be preserved, even if it results in alerts: original behaviour included failing automatically on exception, and ignoring or not using certain test cases.

**Comment by FraPCA** [comment]
https://github.com/Tensa53/commons-net/commit/51e3c07ce75f4bb074561a1ae615ea7d215a8ecf
Reverts the 12 fixes that I added.

**Comment by Tensa53** [comment]
55c35ec
Reverts the 10 fixes that I added

**Comment by Tensa53** [comment]
Hi Francesco. You did a good job on resolving all these smells. In this message i will recap al the smells I resolved.

4918955, c3ad18f, 3a677f0, 8699bef, 142395d, 142395d: Fixed 21/43 of Refactoring a method to reduce its cognitive complexity. 

15a6d77: Fixed 2/6 of Defining a constant instead of duplicating this literal, the third one has been indirectly resolved in commit 3a677f0.

As you early mentioned, I reverted the fixes to the test cases.

One refactoring of the 43 about cognitive complexity has been ignored because it would take too much effort time to resolve it and it wouldn't be compatible with our schedule for the progress of the project
